librubyfmt/src/lib.rs

```diff
@@ -193,6 +194,7 @@ unsafe fn load_ripper() -> Result<(), ()> {
         "../ruby_checkout/ext/ripper/lib/ripper/sexp.rb"
     ))?;
 
+    rb_gc_disable();
```

Comment:
Totally open to suggestions on a better way to do this since I couldn't _really_ figure out why `GC.disable` seemed to be undefined after I bumped to 3, but I ended up just calling the underlying VM implementation here instead to get ~the same result.